### DISCUSSION TITLE
How are the automation rules triggers for "Amount is..." working?

### DISCUSSION THREAD

**Discussion by dklinger** [firefly-iii/firefly-iii#6591] [Firefly III]
Hey. I don't get how the Amount automation rule triggers are working.

For example, If I set i to: "Source account is xyz" AND "Amount is more than..." -> 0,00
It returns withdrawals with an Amount smaller than 0.

If I set it to this:  "Destination account is xyz" AND "Amount is less than..." -> 0,00
It doesn't return any result.

**Comment by JC5** [comment]
The amount is always written as a positive number, even for withdrawals. I'll do some checking later!

**Reply by JC5** [reply]
Amount less than zero will indeed not return anything. This is expected.
